hazelcast/src/test/java/com/hazelcast/cluster/ClusterShutdownTest.java

```diff
@@ -56,11 +57,19 @@ private void testClusterShutdownWithSingleMember(final ClusterState clusterState
         final Thread thread = new Thread(new Runnable() {
             @Override
             public void run() {
-                instances[0].getCluster().shutdown();
+                while (true) {
```

Comment:
This is a distributed shutdown and shutdowns all cluster members. A distributed operation is expected to fail. And also it's clearly defined in its doc. 

```
* @throws IllegalStateException    if member-list changes during the transaction
 * @throws TransactionException     if there's already an ongoing transaction
 *                                  or this transaction fails
 *                                  or this transaction timeouts
```